### ISSUE TITLE
unholy dk not recommending dark transformation

### ISSUE THREAD

**Issue by bitmatt2** [Hekili/hekili#1900]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### WoW Version

Retail (Dragonflight)

### Describe the Issue

while playing on my unholy dk. the addon will recommend dark transformation if left in the primary rotation. however, if i set it as a cooldown to be recommended in the cooldown window; it will never show up

### How to Reproduce

show cooldowns in separate window
set dark transformation to a cooldown under the toggles tab

### Talent Loadout

BwPAdLUHklrTr3hBIcYWdbnCGDAIBSSISEIJRI4AJJRAAAAAAAAAAKQEQSaAgEpINRSSikA

### Snapshot

https://pastebin.com/rJpbmwqJ

### Raidbots Sim Report (Link)

```
# Rockzombie - Unholy - 2022-11-07 12:53 - US/Tichondrius # SimC Addon 10.0.0-1 # WoW 10.0.0.46455, TOC 100000 # Requires SimulationCraft 1000-01 or newer  deathknight="Rockzombie" level=60 race=orc region=us server=tichondrius role=attack professions=alchemy=1/engineering=1 spec=unholy  talents=BwPAU3BjZ/IAySQ6AmAmsjvQ+BAIBSSISEIJRI4AJJRAAAAAAAAAAKQEQSaAgEpINRSSikA  # Saved Loadout: mythic+ # talents=BwPAU3BjZ/IAySQ6AmAmsjvQ+BAIBSSISEIJRI4AJJRAAAAAAAAAAKQEQSaAgEpINRSSikA  covenant=necrolord soulbind=plague_deviser_marileth:4,323074/65:5:0/323091/283:5:0/323081/127:5:0/88:5:0/323095/125:5:0/352109/99:5:0/352110 # soulbind=emeni:5, # soulbind=bonesmith_heirmir:10,326514/284:5:1/326507/99:5:0/326512/108:5:0/68:5:0/326509/127:5:0/350899/283:5:0/350936 # conduits_available=91:5/124:5/127:5/284:5/65:5/79:5/80:5/83:5/89:5/122:5/123:5/125:5/68:5/99:5/106:5/70:5/74:5/86:5/88:5/283:5/108:5/121:5 renown=62  head=,id=171445,bonus_id=6893/7881,drop_level=60,crafted_stats=40/36 neck=,id=173146,gem_id=173130,bonus_id=7880,drop_level=60,crafted_stats=49 shoulder=,id=171447,bonus_id=6893/7880,drop_level=60,crafted_stats=36/40 back=,id=173215,bonus_id=6893/7880,drop_level=60,crafted_stats=49/36 chest=,id=171412,enchant_id=6214,bonus_id=6953/6716/6650/6649/1487/8156 wrist=,id=171449,enchant_id=6222,bonus_id=6893/7880,drop_level=60,crafted_stats=36/49 hands=,id=171444,enchant_id=6210,bonus_id=6893/7880,drop_level=60,crafted_stats=49/36 waist=,id=171448,bonus_id=6893/7880,drop_level=60,crafted_stats=36/40 legs=,id=171446,bonus_id=6893/7880,drop_level=60,crafted_stats=49/32 feet=,id=171443,bonus_id=6893/7880,drop_level=60,crafted_stats=49/32 finger1=,id=173133,enchant_id=6166,gem_id=173130,bonus_id=7880,drop_level=60,crafted_stats=32 finger2=,id=173132,enchant_id=6166,gem_id=173130,bonus_id=7880,drop_level=60,crafted_stats=36 trinket1=,id=173087 trinket2=,id=178808,bonus_id=7359/6652/1576/6646 main_hand=,id=185810,enchant_id=3368,bonus_id=6806/6652/1566/4785  ### Gear from Bags # # Bladesworn Harbinger's Greathelm (197) # head=,id=182250,bonus_id=7201/1514/6646 # # Darkfrost Helmet (197) # head=,id=178777,bonus_id=7254/7359/6652/7194/1511/6646 # # Versatile Solenium Necklace (168) # neck=,id=173144,gem_id=173130,bonus_id=7185,drop_level=60,crafted_stats=36 # # Unyielding Combatant's Pauldrons (184) # shoulder=,id=178802,bonus_id=7362/7359/6652/1498/6646 # # Bladesworn Harbinger's Pauldrons (197) # shoulder=,id=182255,bonus_id=7201/1514/6646 # # Gorewrought Drape (197) # back=,id=181638,bonus_id=6652/1472/5894/6646 # # Drape of Twisted Loyalties (207) # back=,id=180123,bonus_id=7383/7359/6652/1521/6646 # # Cloak of Enveloping Manifestations (187) # back=,id=178851,enchant_id=6208,bonus_id=7377/7359/6652/1501/6646 # # Ashjra'kamas, Shroud of Resolve (155) # back=,id=169223,bonus_id=6601/1472 # # Putrefied Leather Jerkin (120) # chest=,id=176935,bonus_id=6909,drop_level=60 # # Shadowsteel Breastplate (230) # chest=,id=171442,enchant_id=6230,bonus_id=6893/7461,drop_level=60,crafted_stats=40/49 # # Chestplate of the Deadly Coil (190) # chest=,id=171412,enchant_id=6216,bonus_id=6952/6716/6649/6648/1487/8156 # # Bladesworn Harbinger's Chestguard (190) # chest=,id=182251,enchant_id=6214,bonus_id=7200/1507/6646 # # Breastplate of Brutal Dissonance (200) # chest=,id=180099,bonus_id=7255/7359/6652/1514/6646 # # Bladesworn Harbinger's Vambraces (197) # wrist=,id=182257,enchant_id=6222,bonus_id=7201/1514/6646 # # Armguards of Absolute Zero (235) # wrist=,id=171419,enchant_id=6222,gem_id=173130,bonus_id=6946/6758/6650/6648/1532/6935/8156 # # Soldier's Stoneband Wristguards (187) # wrist=,id=183031,bonus_id=7186/6652/7194/1459/6646 # # Distorted Construct's Gauntlets (200) # hands=,id=180104,enchant_id=6210,bonus_id=7367/7359/6652/1514/6646 # # Bladesworn Harbinger's Crushers (197) # hands=,id=182253,enchant_id=6210,bonus_id=7201/1514/6646 # # Fused Bone Greatbelt (187) # waist=,id=178734,bonus_id=7363/7359/6652/7194/1501/6646 # # Bladesworn Harbinger's Warbelt (197) # waist=,id=182256,bonus_id=7201/1514/6646 # # Bladesworn Harbinger's Greaves (197) # legs=,id=182254,bonus_id=7201/1514/6646 # # Magma Plated Legguards (37) # legs=,id=60352 # # Galvanized Oxxein Legguards (184) # legs=,id=178800,bonus_id=7358/7359/6652/1498/6646 # # Bladesworn Harbinger's Stompers (190) # feet=,id=182252,bonus_id=7200/1507/6646 # # Warboots of Ruthless Conviction (213) # feet=,id=180101,bonus_id=7371/7359/6652/1527/6646 # # Muckwallow Stompers (184) # feet=,id=178774,bonus_id=7362/7359/6652/1498/6646 # # Ritual Bone Band (194) # finger1=,id=178870,enchant_id=6170,bonus_id=7379/7359/6652/7193/1508/6646 # # Ritual Bone Band (184) # finger1=,id=178870,enchant_id=6170,bonus_id=7376/7359/6652/7193/1498/6646 # # Versatile Solenium Ring (200) # finger1=,id=173131,enchant_id=6170,gem_id=173129,bonus_id=7183,drop_level=60,crafted_stats=49 # # Leafed Banewood Band (194) # finger1=,id=181708,enchant_id=6170,bonus_id=6652/7194/1472/5891/6646 # # Gluttonous Spike (187) # trinket1=,id=184023,bonus_id=7186/6652/1459/6646 # # Master Duelist's Chit (187) # trinket1=,id=181358,bonus_id=6652/1472/5884/6646 # # Darkmoon Deck: Indomitable (200) # trinket1=,id=173096 # # Unchained Aspirant's Medallion (184) # trinket1=,id=185309,bonus_id=7257/1498/6616 # # Unchained Aspirant's Badge of Ferocity (190) # trinket1=,id=185161,bonus_id=7265/1504/6616 # # Rimebinder's Runeblade (184) # main_hand=,id=178780,enchant_id=3368,bonus_id=7404/7359/1498/6646 # # Duskwarder Maul (190) # main_hand=,id=179537,bonus_id=6652/1472/5887/6646 # # Duskwarder Scimitar (194) # main_hand=,id=179513,bonus_id=6652/1472/5891/6646 # # Ossified Broadaxe (187) # main_hand=,id=184251,enchant_id=3368,bonus_id=7186/6652/1485/6646  # Checksum: e22d88a7
```

### Contact Information

bitmatt2@gmail.com

**Comment by Hekili** [comment]
Your Raidbots sim report was not a link to a completed sim, but we can ignore that.

Your snapshot is not complete.  A snapshot is required to diagnose this issue.

# Snapshot

Please supply a Snapshot of the addon's decision-making when you are seeing this issue in-game. This is not a screenshot. To generate a Snapshot, please complete the following steps.

 * Log into your WoW character.
 * Recreate the issue you are reporting (i.e., generate 5 Holy Power).
 * When you see the recommendation you disagree with, press ALT-SHIFT-P to Pause and Snapshot (or use ALT-SHIFT-[ to Snapshot without pausing).
   * You can change these keybindings in /hekili > Toggles if needed.
   * If you take the snapshot at a different time, it will not be useful and you'll be asked to submit a new one.
 * Type /hekili and press Enter.
 * Open the Snapshots section on the left side.
 * Select the snapshot that shows the recommendation that you disagree with.
   * Snapshots are labeled with your specialization, the display in question (usually, you'll choose Primary), and the recommendations the addon made with their timings.
 * Click in the Export Snapshot box, press CTRL+A to select all text including hidden text and press CTRL+C to copy.
 * Open [https://pastebin.com](https://github.com/Hekili/hekili/issues/Pastebin), use CTRL+V to paste the snapshot text, then click Create New Paste.
 * Provide the Pastebin ***link*** in a new comment.

This step is essential, as most issues are related to specific classes, specializations, gear, talent choices, or other game systems.  If you do not provide this information, I cannot triage your problem.

**Comment by bitmatt2** [comment]
https://pastebin.com/M51B1nji

I hope the link above is the snapshot ur after. the addon should be
recommending dark transformation in cooldowns along with apocalypse but it
is not. my si is the link below. let me know if u need any more info.

https://www.raidbots.com/simbot/report/b8PuBRn6VZVCaJX1D6dkHF

**Comment by Hekili** [comment]
Thanks.  I assume your Ghoul pet was active at this point in time.  Do you have any glyphs applied that change your ghoul?

**Comment by bitmatt2** [comment]
Yes, I do.  Turns him into a giest

**Comment by Hekili** [comment]
This should be resolved in 08.3 (the new release, just posted).
